geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/PutAllClientServerDistributedTest.java

```diff
@@ -2305,24 +2300,21 @@ public void testEventIdMisorderInPRSingleHop() {
     myRegion.getAttributesMutator().addCacheListener(new CountingCacheListener<>(clientCounter));
     myRegion.registerInterest("ALL_KEYS");
 
+    // server1 and server2 will closeCache after created 10 keys
     // server1 add slow listener
     server1.invoke(() -> {
       Region<String, TickerData> region = getCache().getRegion(regionName);
       region.getAttributesMutator()
           .addCacheListener(new SlowCountingCacheListener<>(new Action<>(Operation.CREATE,
-              creates -> executorServiceRule.submit(() -> {
-                closeCacheConditionally(creates, 10);
-              }))));
+              creates -> closeCacheConditionally(creates, 10))));
     });
 
     // server2 add slow listener
     server2.invoke(() -> {
       Region<String, TickerData> region = getCache().getRegion(regionName);
       region.getAttributesMutator()
           .addCacheListener(new SlowCountingCacheListener<>(new Action<>(Operation.CREATE,
-              creates -> executorServiceRule.submit(() -> {
-                closeCacheConditionally(creates, 10);
-              }))));
+              creates -> closeCacheConditionally(creates, 30))));
```

Comment:
It could so happen the number of buckets on this server could be less than 30; making putAll to success, changing it to less number will help.